### DISCUSSION TITLE
support for older versions of directX games?

### DISCUSSION THREAD

**Discussion by F-16Aircraft** [obsproject/obs-studio#7733] [Development Questions] [answered]
hey i noticed that i am unable to get obs to record older windows2000 games that run on DirectX 7-8.1 or later @ full screen and was wondering if its possible for someone to add support for these older windows games so obs can record it properly in 1920x1080?

**Comment by Fenrirthviti** [comment, marked as answer]
Those games don't support the necessary APIs to be compatible with our capture hooks. My loose understanding is that the functions to be able to capture were not included as part of the spec, and would need to be implemented on a per-game basis, so there are a handful of DX8 games that will work, but it's not guaranteed. I'd recommend just using window or display capture in those cases.

**Reply by F-16Aircraft** [reply]
windows capture and display capture will capture the game, but the recording shows the game in the top left of the screen, instead of the center, and the game only plays on forced fullscreen

**Reply by Fenrirthviti** [reply]
At that point, that sounds like a support issue where someone on our Forums or Discord should be able to help.

**Reply by F-16Aircraft** [reply]
i tried going to forums but im blocked from it for some reason, the forum page doesnt load, do you have the discord invite link for obs?

**Reply by Fenrirthviti** [reply]
https://obsproject.com/discord

**Comment by F-16Aircraft** [comment]
discord helped with the answer, when recording old windows games made during windows xp or windows 2000 operating system era click on the source and press "ctrl + F" and it will fit the source to the screen, recording the game in the center of the screen instead of the top left
